Doc/library/dbm.rst

```diff
@@ -149,12 +149,12 @@ supported.
    raised for general mapping errors like specifying an incorrect key.
 
 
-.. function:: open(filename[, flag[, mode]])
+.. function:: open(filename[, flags[, mode]])
```

Comment:
Double-check me, but https://github.com/python/cpython/blob/master/Modules/_gdbmmodule.c#L553 seems to suggest this is actually even more accurate:
```suggestion
.. function:: open(filename[, flags="r"[, mode]])
```